scss/forms/_form-check.scss

```diff
@@ -152,12 +155,14 @@
   padding-left: $form-switch-padding-start;
 
   .form-check-input {
+    --#{$prefix}form-switch-bg: #{escape-svg($form-switch-bg-image)};
+
     position: relative; // Boosted mod: needed because of filter property
     width: $form-switch-width;
     height: $form-switch-width * .5;
     margin-left: $form-switch-padding-start * -1;
     background-color: $white; // Boosted mod
-    background-image: $form-switch-bg-image, var(--#{$boosted-prefix}switch-gradient);
+    background-image: var(--#{$prefix}form-switch-bg), var(--#{$boosted-prefix}switch-gradient); // Boosted mod: extra param `var(--#{$boosted-prefix}switch-gradient)`
```

Comment:
Since we now have CSS var on `.form-check` couldn't we just change the variable instead of reinitialize it, to lighten the bundle ?

Reply:
Rethinking of this, might not be that useful atm.